uber/utils.py

```diff
@@ -240,15 +240,12 @@ def _record_email_sent(email):
 
 
 class Charge:
-    def __init__(self, targets=(), amount=None, description=None, receipt_email=None):
-        self.targets = [self.to_sessionized(m) for m in listify(targets)]
-
-        # performance optimization
-        self._models_cached = [self.from_sessionized(d) for d in self.targets]
```

Comment:
Instead of doing this up front, we use a `@cached` models property, and do it lazily.